src/encoded/schemas/mixins.json

```diff
@@ -717,6 +717,22 @@
             }
         }
     },
+    "preservation_medium": {
```

Comment:
These mixins are not quite formatted correctly.  They need a parent container - see 'protocols' below.  These could be grouped under a single parent eg. 'preservation_info' so when you mix it in both properties come along as you are mixing in the same 2 properties in both 'sample' and 'tissue_sample' schemas.